spark/src/main/scala/org/apache/spark/sql/delta/DeltaLog.scala

```diff
@@ -400,12 +403,8 @@ class DeltaLog private(
         "clientFeatures" -> clientSupportedFeatureNames.mkString(","),
         "clientUnsupportedFeatures" -> clientUnsupportedFeatureNames.mkString(",")))
 
-    val tableNameInException = if (catalogTable.isDefined) {
-      catalogTable.get.identifier.copy(catalog = None).unquotedString
-    }
-    else {
-      dataPath.toString()
-    }
+    val tableNameInException = catalogTable.map(_.identifier.copy(catalog = None).unquotedString)
+      .getOrElse(dataPath.toString())
```

Comment:
nit: It might be slightly easier to read this way?
```suggestion
    val tableNameInException = catalogTable match {
      case Some(ct) => ct.identifier.copy(catalog = None).unquotedString
      case None => dataPath.toString()
    }
```
(your decision)

Reply:
Thanks, I applied the change.